content/2021-11-18-brigade-2-cuts-first-release-candidate.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Brigade 2 Cuts First Release Candidate"
+slug: "2021-11-18-brigade-2-cuts-first-release-candidate"
```

Comment:
Was it intended to date this back in time (yesterday, 11/18).  Or, should it be dated today (11/19)?

Reply:
@vdice amazing catch! Not intentional. I'm just an idiot.

Reply:
Fixed!

Reply:
:shipit: